README.md

```diff
@@ -14,6 +14,8 @@ PREFIX=
 ## Installation
 
 ```bash
+$ git clone https://github.com/EddieHubCommunity/EddieBot.git
```

Comment:
This handles the installation for the usage with the `https`-Link.
Do you think we also need to mention the `ssh` or `github`-cli?

Reply:
I think the https links is enough

Reply:
I think this is enough

Reply:
Personally I prefer and recommend ssh

maybe include the other options?